crates/codegen/src/lib.rs

```diff
@@ -129,30 +133,30 @@ pub struct Generator {
 
 impl Generator {
     /// Create a new [`CodeGenBuilder`].
```

Comment:
Wasn't introduced in this PR but noticed a warning I got when I generated the module docs
```suggestion
    /// Create a new [`Generator`].
```